localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,390 @@
 ﻿l_english:
+ ####################################################
+ # Special titles
+ ### Horde ###
+ warchief_male_emperor:0 "Warchief"
+ warchief_female_emperor:0 "Warchief"
 
-####################################################
-#Special titles
+ ### Scourge ###
+ lich_king_male:0 "Lich King"
+ lich_king_female:0 "Lich Queen"
+
+ ### Night Elves ###
  high_priest_kaldorei_male:0 "$HIGH_PRIEST_MALE_NAME$"
  high_priestess_kaldorei_female:0 "$HIGH_PRIEST_FEMALE_NAME$"
+
+ # Special cultural titles (sorted by culture, then government type).
+ ### Orcs ###
+ # Feudal
+ overlord_male_orc_count:0 "Overlord"
+ overlord_female_orc_count:0 "Overlord"
+ high_overlord_male_orc_duke:0 "High Overlord"
+ high_overlord_female_orc_duke:0 "High Overlord"
+ warlord_male_orc_king:0 "Warlord"
+ warlord_female_orc_king:0 "Warlord"
+ warchief_male_orc_emperor:0 "Warchief"
+ warchief_female_orc_emperor:0 "Warchief"
+
+ ### High Elves / Blood Elves ###
+ # Feudal
+ lord_male_highelf_count:0 "Lord"
+ lady_female_highelf_count:0 "Lady"
+ lordship_highelf_county:0 "Lordship"
+ high_king_male_highelf_emperor:0 "High King"
+ high_queen_female_highelf_emperor:0 "High Queen"
+ high_kingdom_highelf_empire:0 "High Kingdom"
+
+ ### Highbornes / Nightbornes ###
+ # Nightborne specific
+ vicomte_male_nightborne_count:0 "Vicomte"
+ vicomtesse_male_nightborne_count:0 "Vicomtesse"
+ vicomte_nightborne_county:0 "Vicomte"
+
+ # Feudal
+ prince_male_highborne_duke:0 "Prince"
+ princess_female_highborne_duke:0 "Princess"
+ principality_highborne_duchy:0 "Principality"
+ grand_prince_male_highborne_king:0 "Grand Prince"
+ grand_princess_female_highborne_king:0 "Grand Princess"
+ grand_principality_highborne_kingdom:0 "Grand Principality"
+ king_male_highborne_emperor:0 "King"
+ queen_female_highborne_emperor:0 "Queen"
+
+ ### Nagas ###
+ lord_male_naga_count:0 "Lord"
+ lady_female_naga_count:0 "Lady"
+ lordship_naga_county:0 "Lordship"
+ prince_male_naga_duke:0 "Prince"
+ princess_female_naga_duke:0 "Princess"
+ principalty_naga_duchy:0 "Principalty"
```

Comment:
there are some Naga's called prince, but i think it fits better if they are sons to like a Raj and have the title as a successor or something. almost none or none is called princess, almost all are called Lady.
I think for Duchy titles males could be called **Tide Lord** or **Warlord**, and _**High Warlord** can be Kingdom_ title. And for Females i think **Matron** or **Revered Lady**. then the duchy title itself could be called Coildom (becouse coil is a thing?), Tide domain or just duchy or something.

Reply:
It's just to behave like the highbornes.

Reply:
Yes, I get your point but it weird that a Warlord or Tidemistress is higher up then Princess, I don't think Azshara goes around and happily dispute them. If anything i think those should be succession titles only for empire or possible kingdom successors

Reply:
It is only because the nagas are more military than the highbornes.

Reply:
why are you agreeing with me but sounding as you disagreeing with me?

Reply:
exactly, Azshara made her realm more absolutism, rather then having feudal titles, and giving those nobles a lot of power, she rather have the military do things, and having the power by herself, so less nobles with great power distributed and more control herself

Reply:
So i am not disputing that princes and princesses exist, but i don't think they would exist on a duchy level.

Reply:
Not sure I understand right what do you want but eh, I'm tired
So done in a77aec49e